modules/tinymce/src/themes/silver/main/ts/ui/menus/item/structure/ItemSlices.ts

```diff
@@ -88,6 +89,38 @@ const renderDownwardsCaret = (icons: IconProvider): AlloySpec => ({
   }
 });
 
+const renderContainer = (components: Array<AlloySpec>, direction: string): AlloySpec => ({
+  dom: {
+    tag: 'div',
+    classes: [ direction === 'vertical' ? ItemClasses.containerColumnClass : ItemClasses.containerRowClass ]
+  },
+  components
+});
+
+const renderImage = (src: string, width: Optional<string>, height: Optional<string>, alt: Optional<string>): AlloySpec => ({
+  dom: {
+    tag: 'img',
+    classes: [ ItemClasses.imageClass ],
+    styles: {
+      'vertical-align': 'top'
+    },
+    attributes: {
+      src,
+      ...width.map((width) => ({ width })).getOr({}),
+      ...height.map((height) => ({ height })).getOr({}),
+      ...alt.map((alt) => ({ alt })).getOr({})
```

Comment:
I mentioned this on the mentions PR, but without an `alt` this will cause an accessibility issue. So should this just default to using `alt=''` so it's treated as a decorative image?